### DISCUSSION TITLE
How do I stop the giant pink "Meet your new frontend workshop" splash screen from popping up constantly?

### DISCUSSION THREAD

**Discussion by threehams** [storybookjs/storybook#28704] [Help] [answered]
### Summary

I'm doing a major version upgrade from v6 to v8, and decided to start over by using `npx storybook@latest init`.

Once Storybook loads, it's immediately blocked for about 5 seconds by a giant pink splash screen saying "Meet your new frontend workshop." After dismissing, it'll pop up again while navigating around sometimes, and will always pop up again when restarting the server (which I'm doing a lot, since I just initialized it).

What's required to make this go away and stop coming back?

### Additional information

`main.js` file auto-created by `npx storybook init`:

```
/** @type { import('@storybook/nextjs').StorybookConfig } */
const config = {
  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
  addons: [
    "@storybook/addon-onboarding",
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@chromatic-com/storybook",
    "@storybook/addon-interactions",
  ],
  framework: {
    name: "@storybook/nextjs",
    options: {},
  },
  staticDirs: ["../public"],
};
export default config;
```

**Comment by threehams** [comment, marked as answer]
Figured it out. I'm getting confused by `npx storybook@latest init` initializing Storybook, then running it, so I was running the same command again and getting everything re-initialized.

There's still maybe a question about why that splash screen happens multiple times after initialization, but it's not really blocking anything now.

**Comment by nohack-dev** [comment]
check the URL. I think there might be an onboarding-related value in the query, like `localhost:6006?onboarding=true`
